File: Backend/src/main/java/com/web/matcha/web/dto/UserDto.java
package com.web.matcha.web.dto;

import lombok.Getter;
import lombok.Setter;

@Setter
@Getter
public class UserDto {
	private Integer id;

	private String username;

	private String email;

	private String password;

	private String first_name;

	private String last_name;
}

File: Frontend/src/app/data/dto/receive/user.dto.ts
import {ProfileDto} from "./profile.dto";

export class UserDto {
	/**
	 * The user id.
	 */
	userId!: number;

	/**
	 * The username.
	 */
	username!: string;

	/**
	 * The email.
	 */
	email!: string;

	/**
	 * The first name.
	 */
	firstName!: string;

	/**
	 * The last name.
	 */
	lastName!: string;

	/**
	 * The user profile.
	 */
	profile!: ProfileDto;
}